app.js

```diff
@@ -52,4 +52,254 @@ document.addEventListener('DOMContentLoaded', () => {
+    liveButton: 'See Live',
+    liveImg: './image/live-img.png',
+    sourceButton: 'See Source',
+    sourceImg: './img/github.png',
+  }];
+
+  const rem = [{
+    title: '',
+    close: '',
+    skillTag: ['', '', ''],
+    imgSrc: '',
+    summary: '',
+    liveButton: '',
+    sourceButton: '',
+  },
+  {
+    title: '',
+    close: '',
+    skillTag: ['', '', ''],
+    imgSrc: '',
+    summary: '',
+    liveButton: '',
+    sourceButton: '',
+  },
+  {
+    title: '',
+    close: '',
+    skillTag: ['', '', ''],
+    imgSrc: '',
+    summary: '',
+    liveButton: '',
+    sourceButton: '',
+  },
+  {
+    title: '',
+    close: '',
+    skillTag: ['', '', ''],
+    imgSrc: '',
+    summary: '',
+    liveButton: '',
+    sourceButton: '',
+  },
+  {
+    title: '',
+    close: '',
+    skillTag: ['', '', ''],
+    imgSrc: '',
+    summary: '',
+    liveButton: '',
+    sourceButton: '',
+  },
+  {
+    title: '',
+    close: '',
+    skillTag: ['', '', ''],
+    imgSrc: '',
+    summary: '',
+    liveButton: '',
+    sourceButton: '',
+  },
+  {
+    title: '',
+    close: '',
+    skillTag: ['', '', ''],
+    imgSrc: '',
+    summary: '',
+    liveButton: '',
+    sourceButton: '',
+  },
+  ];
+
+  function remove(g, array) {
+    const ject = array[g];
+    content.style.display = 'none';
+    grand.style.display = 'none';
+    heading.innerHTML = ject.title;
+    exit.innerHTML = ject.close;
+    const s = ject.skillTag;
+    const [x, y, z] = s;
+    html.innerHTML = x;
+    bootstrap.innerHTML = y;
+    rails.innerHTML = z;
+    project.src = ject.imgSrc;
+    text.innerHTML = ject.summary;
+    live.innerHTML = ject.liveButton;
+    source.innerHTML = ject.sourceButton;
+  }
+
+  function projects(n, arr) {
+    const pro = arr[n];
+    heading.innerHTML = pro.title;
+    exit.innerHTML = pro.close;
+    function clear() {
+      remove(n, rem);
+    }
+
+    exit.addEventListener('click', clear);
+    const a = pro.skillTag;
+    const [b, c, d] = a;
+    html.innerHTML = b;
+    bootstrap.innerHTML = c;
+    rails.innerHTML = d;
+    project.src = pro.imgSrc;
+    text.innerHTML = pro.summary;
+    live.innerHTML = pro.liveButton;
+    source.innerHTML = pro.sourceButton;
+  }
+
+  function pop0() {
+    projects(0, projectItems);
+    content.style.display = 'grid';
+    grand.style.display = 'block';
+  }
+
+  function pop1() {
+    projects(1, projectItems);
+    content.style.display = 'grid';
+    grand.style.display = 'block';
+  }
+
+  function pop2() {
+    projects(2, projectItems);
+    content.style.display = 'grid';
+    grand.style.display = 'block';
+  }
```

Comment:
- [ ] As requested by the previous reviewer and following the project requirements, your popup should match as much as possible the Figma popup. Actually, you did some progress, however, the style seems to be dissimilar to the template design. Please ensure that your popup has the same `size`, `background-image`, `fonts`, `padding` and `margin` etc.

**Your popup**
![Screenshot from 2022-07-28 22-47-39](https://user-images.githubusercontent.com/80612925/181634703-d76268b7-de38-419e-826c-ea7170feaf71.png)

**Figma popup**
![image](https://user-images.githubusercontent.com/80612925/181634767-fe33190b-6d4b-4270-af0e-69b8a496c04d.png)

**_Note: Check the background, buttons icons, margin, padding, popup size, font-size and font-family, and element alignment._**